queue/command_queued.go

```diff
@@ -0,0 +1,21 @@
+package queue
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"github.com/micromdm/micromdm/queue/internal/commandqueuedproto"
+)
+
+func MarshalQueuedCommand(udid, uuid string) ([]byte, error) {
```

Comment:
I'd prefer to keep Marshal, Unmarshal functions consistent across different types and packages. That means they should all have a signature like 

```
type Foo struct{}

MarshalFoo(*Foo) ([]byte, error)
UnmarhalFoo(data []byte, into *Foo) error
```

As long as we keep the methods consistent everywhere it will be easier to develop.